### DISCUSSION TITLE
Stop AR Object from Moving Because of Custom HTML Buttons

### DISCUSSION THREAD

**Discussion by byanraff** [google/model-viewer#4673] [Q&A]
Hi, I have created a custom range inputs to be used on top of AR mode. The issue is whenever the range is overlayed on top of AR models, the model is moving when I swipe the range input (just like when the model is moved when swiped). Is there a possible solution to stop the AR model from moving when I use the HTML range input?
![WhatsApp Image 2024-02-07 at 09 45 52](https://github.com/google/model-viewer/assets/111570157/fb0f3aea-6774-4bf6-954a-7877fad3e75a)

**Comment by elalish** [comment]
Yes, you're looking for
```
document.querySelector(".slider").addEventListener('beforexrselect', (ev) => {
    // Keep slider interactions from affecting the XR scene.
    ev.preventDefault();
  });
```
From https://modelviewer.dev/examples/augmentedreality/index.html
